mgmtsystem/scvmm.py

```diff
@@ -267,12 +266,12 @@ def clone_vm(self, vm_source, vm_host, path, vm_name):
     def does_vm_exist(self, vm_name):
         result = self.run_script("Get-SCVirtualMachine -Name \"{}\" -VMMServer $scvmm_server"
             .format(vm_name)).strip()
-        return len(result) > 0
+        return True if len(result) else False
```

Comment:
You can replace this by:
```python
return bool(len(result))
```

but essentially, an empty string is the same as empty list, therefore you can do it as follows:
```python
return bool(result)
```